spec/uploaders/logo_svg_uploader_spec.rb

```diff
@@ -0,0 +1,56 @@
+require "rails_helper"
+require "carrierwave/test/matchers"
+require "exifr/jpeg"
+
+describe LogoSvgUploader, type: :uploader do
+  include CarrierWave::Test::Matchers
+
+  let(:image_svg) { fixture_file_upload("300x100.svg") }
+  let(:image_webp) { fixture_file_upload("800x600.webp", "image/webp") }
+
+  # we need a new uploader before each test, and since the uploader is not a model
+  # we can recreate it quickly in memory with `let!`
+  let!(:uploader) { described_class.new }
+
+  before do
+    described_class.include CarrierWave::MiniMagick # needed for processing
+    described_class.enable_processing = true
+  end
+
+  after do
+    described_class.enable_processing = false
+    uploader.remove!
+  end
+
+  it "stores files in the correct directory" do
+    expect(uploader.store_dir).to eq("uploads/logos/")
+  end
+
+  describe "error handling" do
+    it "raises a CarrierWave error which can be parsed if MiniMagick timeout occurs" do
+      allow(MiniMagick::Image).to receive(:new).and_raise(Timeout::Error)
+
+      expect { uploader.store!(image_svg) }.to raise_error(CarrierWave::IntegrityError, /Image processing timed out/)
+    end
+  end
+
+  describe "processed images" do
```

Comment:
(blocking): These tests are failing on Travis with a MiniMagick error: 
```
Failed to manipulate with MiniMagick, maybe it is not an image? Original Error: `convert /home/travis/build/forem/forem/tmp/1639126885-827978499212382-0004-7279/300x100.svg -auto-orient /tmp/image_processing20211210-10742-7n7kz8.png` failed with error:
       convert-im6.q16: unable to read font `helvetica' @ error/annotate.c/RenderFreetype/1338.
       convert-im6.q16: non-conforming drawing primitive definition `text' @ error/draw.c/RenderMVGContent/4300.
```

but passing locally.

Reply:
Hmm, try changing the following context on rails helper
```
-  config.fixture_path = "#{::Rails.root}/spec/fixtures"
+  config.fixture_path = "#{::Rails.root}/spec/fixtures/files"
```